sway-lib-std/src/lib.sw

```diff
@@ -12,7 +12,7 @@ dep constants;
 dep contract_id;
 dep context;
 dep hash;
-dep r#storage;
+dep storage;
```

Comment:
Should the storage be changed in this issue? I think if that is a problem then it should be its own issue which is merged into master and then master is merged into this. Why was it changed in the first place?

Reply:
nope, I actually just reverted this change as it's not up to this PR to touch it.
Not sure why it was changed, I certainly didn't do it intentionally, so maybe fmt did it?

Reply:
CI fails because storage is a keyword and forbidden as a library name. Revert this change to fix.